cli/tests/upload.rs

```diff
@@ -138,7 +154,7 @@ async fn upload_bundle() {
     let time_since_junit_modified = chrono::Utc::now()
         - chrono::DateTime::from_timestamp_nanos(bundled_file.last_modified_epoch_ns as i64);
     assert_eq!(time_since_junit_modified.num_minutes(), 0);
-    assert!(bundled_file.owners.is_empty());
+    assert_eq!(bundled_file.owners, ["@user"]);
```

Comment:
This is the CODEOWNER for the JUnit.XML file... not any single test referenced by the JUnit.XML file.